Library/Formula/crystal.rb

```diff
@@ -0,0 +1,22 @@
+class Crystal < Formula
+  desc "Code generator for every language, library and framework"
+  homepage "http://crystal.sh"
+  url "https://github.com/crystal/crystal/archive/v0.5.1.tar.gz"
+  sha256 "6c022ad59a10cec58856dba858a030b745333909108c08bd326c20ac0cb144fb"
+
+  depends_on "node"
+
+  def install
+    libexec.install Dir["*"]
+    bin.install_symlink Dir["#{libexec}/bin/*"]
+  end
+
+  def post_install
+    cd libexec
+    system "npm", "install"
```

Comment:
`superenv` punts `npm` out of the `$PATH`, which does cause build failure during `def install`.

So far we've worked around it by either manually re-adding `HOMEBREW_PREFIX/"bin"` to the `$PATH` or by handling it in `post_install`.

Personal point of view is that we should be using `ENV.prepend_path "PATH", "#{Formula["node"].opt_libexec}/npm/bin"` instead in applicable formulae (Or doing that automatically).